### PR TITLE
Fix appveyor ci for round4

### PR DESCRIPTION
**Pull request by cosmo0920**
**Which issue(s) this PR fixes**: 

Related to #2854.

* Fix expected position in posfile
* Use correct error message on connecting error
* Sort by tag name instead of timestamps which are same integer value (this causes unstable result)

**What this PR does / why we need it**: 
We should take care of AppVeyor CI result.

**Docs Changes**:
No needed.

**Release Note**: 

None.

### CODE REVIEW

**Comment by cosmo0920** [comment]
Phew, we got green on AppVeyor! https://ci.appveyor.com/project/fluent/fluentd/builds/31285183

File: test/plugin/in_tail/test_position_file.rb [hunk 1]
```diff
@@ -196,7 +196,11 @@ def build_files(file)
       @file.seek(0)
       lines = @file.readlines
       assert_equal 2, lines.size
-      assert_equal "valid_path\t000000000000000a\t000000000000000b\n", lines[0]
+      if Fluent.windows?
+        assert_equal "valid_path\t0000000000000000a\t000000000000000b\n", lines[0]
```
Comment (ganmacs) [inline test/plugin/in_tail/test_position_file.rb]: [question] Why it happens?

Comment (cosmo0920) [reply test/plugin/in_tail/test_position_file.rb]: I'll dig it.

Comment (cosmo0920) [reply test/plugin/in_tail/test_position_file.rb]: Aha! I got it.
In Windows, it is also needed to use binmode for Tempfile! :punch:

**Comment by cosmo0920** [comment]
I'll merge this PR after finishing CIs. :muscle:

**Pull request merged**